public/index.html

```diff
@@ -109,11 +103,15 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-    <script
-      type="text/javascript"
-      src="https://cdn.fromdoppler.com/doppler-ui-library/%REACT_APP_DOPPLER_UI_LIBRARY_VERSION%/js/app.js"
-    ></script>
     <script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=d551604b-d03b-44e4-9ba6-7fdc92f56cb9"></script>
     <script src="https://cdn.fromdoppler.com/loader/v1/loader.js"></script>
   </body>
+  <script type="text/javascript">
+    window["style-guide-configuration"] = {
+    autoInitialize: true,
+  };
+    new AssetServices().load(
+      "%REACT_APP_STYLE_GUIDE_MANIFIEST_FILE%"
+    );
+  </script>
```

Comment:
@GusBaamonde @Cromeror what do you think about moving the reference to the loader and this block of code to the header?

Reply:
I agree, moving this script to the <head> will be available earlier in the body.